packages/parity-electron/src/checkClockSync.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+const { time } = require('sntp');
```

Comment:
In my previous comment, I was actually referring to using imports instead of requires.